### DISCUSSION TITLE
Error Checking validity of types

### DISCUSSION THREAD

**Discussion by ebennaga** [WalletConnect/walletconnect-monorepo#3461] [Developer Support] [answered]
Hi.

I got error of checking validity of types after npm run build.  I am using nextjs for my applicaiton. what should i do to fix it?

**Comment by glitch-txs** [comment]
Try adding `"SkipLibCheck":true` inside the tsconfig.json file

**Reply by ebennaga** [reply]
Yes, i have been added it to tsconfig.json but it doesn't work

**Reply by glitch-txs** [reply]
can you share a small example of the issue? is your project public?

**Reply by ebennaga** [reply]
![bug](https://github.com/WalletConnect/walletconnect-monorepo/assets/2337413/5d2e88a4-b5a9-4758-b02a-b467ed0af6ad)

i got this error after npm run build.

**Reply by glitch-txs** [reply]
make sure you have the latest version of wagmi, I'd suppest opening an issue on their side too

Also check https://wagmi.sh/react/typescript

**Reply by ebennaga** [reply, marked as answer]
thanks dude, i solved it
